blog/2020-10-23-the-making-of-new-user-sdk-tutorials.md

```diff
@@ -0,0 +1,110 @@
+
+## Picking a path
+
+<img alt="image" class="docs-image-centered" src="https://raw.githubusercontent.com/temporalio/documentation-images/main/static/temporal-server-and-sdk-icons.png" />
+
+At a high level, there are two parts to Temporal: the backend and a client-side SDK. Configuring, deploying, and operating the Temporal backend for a live environment is no small task. On the other hand, it is really easy to [get Temporal running on your local machine](/docs/install-temporal-server) in a Docker container. In fact, you can do it with just two terminal commands.
+
+The Docker route definitely simplifies running the backend, which means the majority of friction for new users comes from our SDKs ([Go](https://github.com/temporalio/sdk-go), [Java](https://github.com/temporalio/sdk-java)). While an SDK is meant to abstract the complexities of interacting with the server API, Temporal flips a lot of the preconceived notions of modern application development on their head. The SDK docs needed to do more than just provide example usage. They also needed to show why, and allow the user to grasp the concepts that Temporal is promoting. So we went about scoping something that we could realistically accomplish within that time frame and still be relatively effective.
+
+We decided that the best thing for us to focus on was a great first experience for new users. We wanted something that would enable a developer to start using the product right away but also instill an understanding of the value it provides. We wanted to cultivate that "aha!" moment.
+
+We started as most might, by trying to envision what the ideal new user experience would look like. We then identified as many of the steps it would take to get there as possible. Looking back, I would contend that we managed to lock onto three ideas that we thought would get us closer to the ideal experience. The hope was that once these three ideas were combined they would result in a set of effective SDK tutorials for new users.
+
+## Snipsync
+
+<img alt="image" class="docs-image-centered docs-image-max-width-20" src="https://raw.githubusercontent.com/temporalio/documentation-images/main/static/sync.png" />
+
+It was around this time (early September), that I was testing out a NodeJS tool I had built to improve the experience of creating and maintaining documentation. It downloads Github repos, scrapes code snippets that exist between specific comment wrappers, and writes the snippets to their corresponding comment wrappers in Markdown files. We borrowed the idea from [Google's proprietary version](https://github.com/GoogleCloudPlatform/golang-samples/blob/master/secretmanager/get_secret.go#L17) they use for their [Google Cloud documentation](https://cloud.google.com/docs). The concept is fairly simple, and I am still surprised that there was no existing open-source solution. So we made one!
```

Comment:
I think this section is far better but "commit wrapper" is not a familiar term (at least to me).

Reply:
I think showing a very concise example would go a long way.